src/GovUk.Education.ExploreEducationStatistics.Content.ViewModels/ReleaseSearchViewModel.cs

```diff
@@ -28,7 +28,7 @@ public ReleaseSearchViewModel(
         ReleaseVersionId = releaseVersion.Id;
         Published = releaseVersion.Published ?? throw new ArgumentException("Release must have a published date");
         PublicationTitle = publication.Title;
-        Summary = ExtractHtmlBlocks(releaseVersion.SummarySection.Content).StripHtml();
+        Summary = publication.Summary.StripHtml();
```

Comment:
PublicationCacheViewModel.Summary should be plain text, not html, so no need for `.StripHtml()`.